deploy/helm/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for ndm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+release:
+  version: "0.9.0"
```

Comment:
nit: update to latest release tag ?

Reply:
latest is now 0.9.1, there was a dot release last week for a bug fix.